generate/beat/{{cookiecutter.beat}}/main.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 func main() {
-	err := beat.Run("{{cookiecutter.beat}}", "", beater.New)
+	err := beat.Run("{{cookiecutter.beat}}", {{cookiecutter.beat}}Version, beater.New)
```

Comment:
Do you want a way for the developers of a community  beat to select whether the community beat version should be the same as libbeat or its own version ?

If so, how can we decide the behavior?
For instance, I can add a question when generating the community beat from cookiecutter:

Select versionning:
1 - Same as libbeat
2 - Application specific


Answer [1]  generates `beat.Run("{{cookiecutter.beat}}", "", beater.New)`
Answer [2] generates `beat.Run("{{cookiecutter.beat}}", appVersion, beater.New)`